### PR TITLE
Local docker build

### PR DESCRIPTION
**Pull request by Q-back**
1. upgrade pip like: `RUN pip install --index-url=https://pypi.python.org/simple/ --upgrade pip`
2. `scapy-real` has been renamed to `scapy`
3. `lz4` can't resolve dependency `pytest-runner`, so bumped Ubuntu to 18.04 (as 12.04 didn't work)
4. bumped `mitmproxy` to 13.0 to prevent `x86_64-linux-gnu-gcc` error
5. added `Tornado 4.5` as it's required by `mitmproxy` in this version (higher versions causes errors)
6. changed `libssl-dev` to `libssl1.0-dev` and added `nodejs`, `nodejs-dev`, `node-gyp` and `npm` dependencies
7. Renamed `docker-build.sh` to `docker-build-release.sh` as this script publishes docker image to docker-hub as official release
8. Added `docker-build-local.sh` to allow users to build image locally if they wish to do so.

### CODE REVIEW

File: extras/docker/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # w3af.org
 # https://github.com/andresriancho/w3af/tree/master/extras
 
-FROM ubuntu:12.04
+FROM ubuntu:18.04
```
Comment (andresriancho) [inline extras/docker/Dockerfile]: Thanks, we needed this!

**Comment by andresriancho** [comment]
giphy

Congrats on the first approved PR!

**Comment by SamBalg** [comment]
I am trying to build the image locally , from w3af-develop. The dockerfile is trying to run the dependancy installer but there is no  /tmp/w3af_dependency_install.sh in the code. That is throwing an error when I'm building the image locally.

**Comment by Q-back** [comment]
Hello @SamBalg. Please make sure you're using `develop` branch and you are building w3af with `w3af/extras/docker/docker-build-local.sh` script.

**Pull request merged**